config/manifests/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
```

Comment:
I believe this file is not used anymore.

Reply:
It automatically gets generated when running `make bundle`